packages/zora/src/Zora.ts

```diff
@@ -126,6 +127,17 @@ export const mint = async (
       tokenId,
     })
   }
+  if (referral) {
+    const referralAddress = getAddress(referral)
```

Comment:
This will always return check summed - I'd add a callout since elsewhere we consistently use lower case

Reply:
Removed this line since using a checksummed address is not required anymore.

I was using an array operator previously for the `rewardsRecipients`, which required a checksummed address.